Src/Campus.Master.API/UI/src/app/modules/calendar/event-modal/delete-event/delete-event.component.html

```diff
@@ -3,23 +3,23 @@
     <div class="dialog__header_wrapper">
       <div class="dialog__ico"></div>
       <div>
-        <h1 class="dialog__title">Видалити подію</h1>
-        <p class="dialog__subtitle">Обрати дію</p>
```

Comment:
Consider removing this p element at all in future releases, because of redundant text